epochStart/bootstrap/disabled/disabledShardCoordinator.go

```diff
@@ -0,0 +1,60 @@
+package disabled
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+)
+
+type shardCoordinator struct {
+	numShards uint32
```

Comment:
no need to redefine shardcoordinator. you can use a temporary shardcoordinator where you are meta. just by making a new instance of shardcoordinator.

Reply:
done